services/uploads/package.json

```diff
@@ -8,6 +8,5 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "",
-  "license": "CC0-1.0",
-  "devDependencies": {}
+  "license": "CC0-1.0"
```

Comment:
Expecting `  "packageManager": "yarn@4.12.0",` in the corresponding package.json